### PR TITLE
Cu 3075b6m create a real footer with navigation 

### PR DESCRIPTION
**Pull request by BenDanielGreep**
## Objective
What: Redesign the footer
Why: It was a image before
How: Using @matthew1809's footer from CT
Usage: Main.jsx

## Type

- [ ] Bug Fix
- [ ] New Feature
- [ ] Performance Tweaks
- [X] Style Tweaks
- [X] Code Refactoring
- [ ] Documentation
- [ ] Tests

## Tested

Locally, will test on netlify

## Documented

- [ ] Have you documented in changed file using comments
- [ ] Have you added instructions to the README if needed?

### CODE REVIEW

**Comment by diegolarraz** [comment]
Task linked: [CU-3075b6m Create a real footer with Navigation](https://app.clickup.com/t/3075b6m)

**Comment by matthew1809** [comment]
Screenshot 2022-10-05 at 17 22 33

Would suggest a top border as seen here, and a tad more spacing to make it feel more comfortable. 

Know I didn't do this, but centring the trademark statement similar to picture wouldn't hurt either.

Those are personal points of view, the code is great!

File: src/Main.jsx [hunk 1]
```diff
@@ -137,8 +137,10 @@ export const Main = () => {
               }
             />
           </Routes>
+          {/* NB disabled logic to render footer */}
           {/* To avoid CLS, load in the footer after the carousels render */}
-          {carouselLoaded && <Footer />}
+          {/* {carouselLoaded && <Footer />} */}
```
Comment (matthew1809) [inline src/Main.jsx:142]: Why is this left in at all?

**Review by matthew1809 — COMMENTED** [review]
GG @BenDanielGreep!

File: src/Main.jsx (hunk 1 above)
Comment (BenDanielGreep) [reply src/Main.jsx:142]: Left in because I would eventually like to re-add this functionality
I should've commented

**Review by HugoWith — CHANGES_REQUESTED** [review]
Hey Ben, 

First of all thanks for handling that, it's clean 👏

Few comments if you have the time, if you don't please let me know I can help with that 👌

#1 On mobile view : if possible align to the top or to the center the three blocks

Screenshot 2022-10-05 at 19 04 39

#2 Small responsive issue, everything is cluttered

Screenshot 2022-10-05 at 19 04 18

#3 Align to top or stretch the first block to balance the left side of the footer

Screenshot 2022-10-05 at 19 02 04

#4 It would be super nice to have a real navigation on this footer. We could use the navigation element from the navbar, we could also open the cart for examples. Among fake button of course 😊

#5 We already talk about this earlier but the html and the CSS are a bit complexe for what it does (I know it's legacy code from something made with bootsrap). If you have the time and the energy it would be great to simplify everything and use BEM synthaxe, using rem, % and not only px. But totally understand if you can't, we have a perfectly functional footer already so well done 😊

**Comment by BenDanielGreep** [comment]
@HugoWith implemented the styling  changes requested.
Regarding the legacy code - I've added BEM where I adjusted but I think it would be better to put the full rebuild into another ticket. Happy to take it when we have the bandwidth.

**Comment by BenDanielGreep** [comment]
@HugoWith yes I would include them both in a single ticket, do you agree?

**Pull request closed without merging**